src/main/scala/strawman/collection/Seq.scala

```diff
@@ -26,6 +31,9 @@ trait SeqOps[+A, +CC[X], +C] extends Any
   with ArrayLike[A]
   with Equals {
 
+  protected[this] def c: C
+  protected[this] def seq: Seq[A]
```

Comment:
I didn't check all the `c` uses, but the `seq` uses don't obviously need to be a `seq` as far as I can tell.  None of them seemed to rely upon fast random access, and anyway, `List` doesn't have that.

Anyway, if we really do need this functionality, we should make it public as `toSeq`.  There is no case I can think of where we need it to implement methods, but we can be sure that nobody else needs it.